pytorch_lightning/callbacks/__init__.py

```diff
@@ -1,7 +1,8 @@
-from .pt_callbacks import EarlyStopping, ModelCheckpoint, GradientAccumulationScheduler
+from .pt_callbacks import EarlyStopping, ModelCheckpoint, ReduceLROnPlateauScheduler, GradientAccumulationScheduler
```

Comment:
it looks like a relative import which we shall not use... :)

Reply:
Relative imports of `EarlyStopping`, `ModelCheckpoint` etc. are taken from the original repository. Why is relative import of `ReduceLROnPlateauScheduler` inappropriate?

Reply:
https://github.com/williamFalcon/pytorch-lightning/pull/402#issuecomment-544704568

Reply:
#402 never fixed relative imports in callbacks __init__ as well as in many other places. I'd say that above comment is out of scope for this PR. @borda it might be better to create a separate PR that will properly fix relative imports.

Reply:
It was not that the PR was fixing relative imports, but I tried to make them which was stopped...

Reply:
let's do a separate PR for relative imports.

Reply:
I have opened ticket https://github.com/williamFalcon/pytorch-lightning/issues/459